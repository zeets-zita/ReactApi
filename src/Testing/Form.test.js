import React from 'react';
import Form from '../Components/Form';
import renderer from 'react-test-renderer';


test('renders correctly', () => {
    const tree = renderer
    .create(<Form/>)
    .toJSON();
    expect(tree).toMatchSnapshot();
    });

