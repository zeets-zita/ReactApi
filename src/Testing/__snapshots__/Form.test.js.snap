// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<form>
  <input
    name="city"
    placeholder="City..."
    type="text"
  />
  <button>
    Get Weather
  </button>
</form>
`;
